.github/workflows/test-presto.yml

```diff
@@ -42,11 +42,22 @@ jobs:
             - 16379:6379
       steps:
       - uses: actions/checkout@v2
+      - name: Check if python changes are present
+        id: check
+        env:
+          GITHUB_REPO: ${{ github.repository }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        continue-on-error: true
+        run: |
+          ./scripts/ci_check_python_changes.sh
+          ./scripts/ci_check_python_test_changes.sh
```

Comment:
Can we maybe parameterize this script to avoid creating two largely identical files?

Reply:
> > Anyway, I still don't think you can skip `test/*.py` for E2E tests.
> 
> are we? I'm not seeing any changes to `superset-e2e.yml`

I wanted to keep the scope of this PR as small as possible, hence didn't het add the E2E changes. Once this is merged I'll add similar logic to frontend and E2E tests.

Reply:
Another benefit of parameterizing this is to allow more advanced skipping in the future---for example, we may split/skip E2E tests based on product areas.